ui/demo/less/reader.less

```diff
@@ -60,32 +60,17 @@ body {
 
 
 .reader__popover__bbox {
```

Comment:
Not related to this ticket, but I think the places where we use `bbox` in the class name are causing confusion since it's inconsistent with `bounding-box` used below... it could be beneficial to take a look at these class names while you're doing that other CSS prefix ticket that we discussed in planning yesterday. Just get everything cleaned up/tested/done all at once.